store/tikv/txn_committer.go

```diff
@@ -110,29 +110,29 @@ func (c *txnCommitter) iterKeys(keys [][]byte, f func(batchKeys) error, sizeFn f
 	}
 
 	if firstIsPrimary {
-		err = c.doBatches(batches[:1], f)
+		err = c.doBatches(bo, batches[:1], f)
 		if err != nil {
 			return errors.Trace(err)
 		}
 		batches = batches[1:]
 	}
 	if asyncNonPrimary {
 		go func() {
-			c.doBatches(batches, f)
+			c.doBatches(bo, batches, f)
```

Comment:
@zimulala Seems we don't have to fork here. The original goroutine will not use `bo` any more, so it should be safe.